style_test.go

```diff
@@ -15,29 +15,29 @@ func TestUnderline(t *testing.T) {
 	}{
 		{
 			NewStyle().Underline(true),
-			"\x1b[4;4ma\x1b[0m\x1b[4;4mb\x1b[0m\x1b[4m \x1b[0m\x1b[4;4mc\x1b[0m",
+			"\x1b[4;4ma\x1b[m\x1b[4;4mb\x1b[m\x1b[4m \x1b[m\x1b[4;4mc\x1b[m",
 		},
 		{
 			NewStyle().Underline(true).UnderlineSpaces(true),
-			"\x1b[4;4ma\x1b[0m\x1b[4;4mb\x1b[0m\x1b[4m \x1b[0m\x1b[4;4mc\x1b[0m",
+			"\x1b[4;4ma\x1b[m\x1b[4;4mb\x1b[m\x1b[4m \x1b[m\x1b[4;4mc\x1b[m",
 		},
 		{
 			NewStyle().Underline(true).UnderlineSpaces(false),
-			"\x1b[4;4ma\x1b[0m\x1b[4;4mb\x1b[0m \x1b[4;4mc\x1b[0m",
```

Comment:
why remove the reset?

Reply:
We're not removing them, `\x1b[0m` is the same as `\x1b[m`. One less byte to reset the colors.

Reply:
ouuuuuu ok thank you!!